### PR TITLE
chore(react): Release react wrappers & move build/demo scripts into 1 file

### LINKED ISSUE #6: React wrappers

### PR DESCRIPTION
**Pull request by theiliad**
### Updates
- Updates react & storybook versions
- Makes react package public
- Moves all build/demo scripts out of travis script into a separate `.sh`

Closes #6

### CODE REVIEW

File: packages/react/src/base-chart.js [hunk 1]
```diff
@@ -9,9 +9,4 @@ export default class BaseChart extends React.Component {
 
 		Object.assign(this, this.chart);
 	}
-
-	// TODO
-	// componentWillUnmount() {
-	// 	this.chart.removeChart();
-	// }
```
Comment (theiliad) [inline packages/react/src/base-chart.js]: React takes care of this since the chart renders inside a the component container

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,4 @@ packages/core/demo/bundle/
 packages/react/build/
 storybook-dist
 package-lock.json
+.cache/
```
Comment (zvonimirfras) [inline .gitignore]: Missing newline at the end of file.
```suggestion
.cache/

```

File: packages/core/package.json [hunk 3]
```diff
@@ -82,4 +84,4 @@
       "url": "https://github.com/theiliad"
     }
   ]
-}
+}
```
Comment (zvonimirfras) [inline packages/core/package.json]:
```suggestion
}

```

File: scripts/build-packages-and-demos.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+cd ..
+rm -rf demo
+
+# Build React demos
+cd ../react
+npm run build-storybook
+cp -a storybook-dist/. ../../pages/react
+
+# Go back to project root folder
+cd ../..
```
Comment (zvonimirfras) [inline scripts/build-packages-and-demos.sh]:
```suggestion
cd ../..

```

**Pull request merged**